quickstart/201-iot-hub-with-device-provisioning-service/main.tf

```diff
@@ -0,0 +1,165 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  location = var.resource_group_location
+  name     = random_pet.rg_name.id
+}
+
+# Create storage account & container
+resource "random_string" "sa_name" {
+  length = 12
+  special = false
+  upper = false
+}
+
+resource "azurerm_storage_account" "sa" {
+  name                     = random_string.sa_name.id
+  resource_group_name      = azurerm_resource_group.rg.name
+  location                 = azurerm_resource_group.rg.location
+  account_tier             = "Standard"
+  account_replication_type = "LRS"
+}
+
+resource "azurerm_storage_container" "my_terraform_container" {
+  name                  = "mycontainer"
+  storage_account_name  = azurerm_storage_account.sa.name
+  container_access_type = "private"
+}
+
+
+# Create an Event Hub & Authorization Rule
+resource "random_pet" "eventhubnamespace_name" {
```

Comment:
Could we rename this resource to `eventhub_namespace_name` to maintain a consist naming convention for this module?